src/main/java/clasess/Main.java

```diff
@@ -0,0 +1,241 @@
+/*
+ * Copyright (c) 2022
+ * The main program service class
+ * runs a farm simulator
+ * use it for free
+ */
+
+package clasess;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import java.util.Random;
+import java.util.Scanner;
+
+public class Main {
+    private static final Logger log = LogManager.getLogger(Main.class.getName());
+    static int maxTurnTime = 20;
+    static int maxAnimalBuy = 5;
+    static Random random = new Random();
+    static Scanner input = new Scanner(System.in);
+
+    public static void main(String[] args) {
+        Random random = new Random();
+        int totalTime = 0;
+        int turnTime;
+        Farm myFarm = new Farm();
+        boolean game = true;
+        Animal animal;
+        myFarm.plantField(myFarm.getWheatField(), myFarm.getCornField().getPlantedArea());
+        myFarm.plantField(myFarm.getCornField(), myFarm.getWheatField().getPlantedArea());
+        /* Farm loop */
+        while (game) {
```

Comment:
This loop should be a static method of a Service class,
so here it should be smth like
`FarmService.start();`